apps/docs/components/MDX/storage_management.mdx

```diff
@@ -0,0 +1,109 @@
+If you repeatedly upload profile photos, you will observe that all uploads will accumulate
+in the `avatars` bucket because of their random names with only the latest being referenced
+from `public.profiles` and the others getting orphaned.
+
+In order to automatically remove obsolete storage objects, we can extend the database
+triggers. It is important to note that it is not sufficient to delete the objects from the
+`storage.objects` table because that would orphan and leak the actual storage objects in
+the S3 backend. Instead, we'll need to invoke the storage API itself from within Postgres
+via the `http` extension. So, enable that for the `extensions` schema in the
+Database/Extensions dashboard.
+
+Then, define the following SQL functions in the SQL Editor which allow to easily delete
+storage objects via the API:
```

Comment:
```suggestion
If you upload additional profile photos, they'll accumulate
in the `avatars` bucket because of their random names with only the latest being referenced
from `public.profiles` and the older versions getting orphaned.

To automatically remove obsolete storage objects, extend the database
triggers. Note that it is not sufficient to delete the objects from the
`storage.objects` table because that would orphan and leak the actual storage objects in
the S3 backend. Instead, invoke the storage API within Postgres via the `http` extension. 

Enable the [http extension for the `extensions` schema](https://app.supabase.com/project/_/database/extensions) in the Dashboard.
Then, define the following SQL functions in the SQL Editor to delete
storage objects via the API:
```